pkg/assert/host.go

```diff
@@ -9,17 +9,17 @@ import (
 	. "github.com/onsi/gomega"
 )
 
-// CheckComponentCmdHost runs a command on the host and asserts that the value received contains the specified substring
+// CheckCmdHost runs a command on the host and asserts that the value received contains the specified substring
```

Comment:
ExecCmdOnHost or RunCmdOnHost

Reply:
this func is calling the RunCmd.....to check some result.